README.md

```diff
@@ -48,6 +48,16 @@ npm start
 
 and then go to [http://localhost:3000]().
 
+## Using it via Docker
+
+Run:
+
+```bash
+docker run -it -p 8000:80 asyncapi/studio
+```
+
+and then go to [http://localhost:8000]().
```

Comment:
We have bug here 😅 

```suggestion
and then go to [http://localhost:8000](http://localhost:8000).
```

Could you also fix link (in the 49 line) in the dev section? I don't know that we have broken link here:

```
and then go to [http://localhost:3000](http://localhost:3000).
```

Reply:
Also, could you point in this section to the Docker Image docs (https://registry.hub.docker.com/_/nginx/)? People should know about customization of this image, how to pass custom nginx config or add more volumes, e.g:

```md
To learn how to e.g. pass a custom `nginx` configuration or plug in additional volumes please refer to the [documentation of the `nginx` docker image](https://registry.hub.docker.com/_/nginx/). 
```

Reply:
Oh sorry, I blindly copy-pasted 😬

Please see https://github.com/asyncapi/studio/pull/240/commits/0730b3c9946865d2f4a99f958103dade0c9fc658.

Reply:
Fast! 😄  Thanks very much, everything is ok, I will check again locally that image and we can merge it! :)

Reply:
@fgreinacher We missed about most important part 😅  Could you pass that scripts https://github.com/asyncapi/server-api/blob/master/package.json#L38-L39 to the package.json `scripts`? Of course please change the `server-api` to the `studio`. `semantic-release-docker` plugin needs builded image, it doesn't built it itself.